apps/backoffice-v2/src/Router/Router.tsx

```diff
@@ -114,7 +116,26 @@ const router = createBrowserRouter([
                     children: [
                       {
                         path: '/:locale/transaction-monitoring/alerts/:alertId',
-                        element: <TransactionMonitoringAlertsAnalysisPage />,
```

Comment:
What happened to the non-business drawer?

Reply:
Its mistake.